.ci/windows-steps.yaml

```diff
@@ -29,7 +29,8 @@ steps:
 - bash: |
     git clone --depth 1 https://github.com/mlpack/jenkins-conf.git conf
     
-    curl http://masterblaster.mlpack.org:5005/armadillo-8.400.0.tar.gz | tar xvz
+    curl -O http://masterblaster.mlpack.org:5005/armadillo-8.400.0.tar.gz -o armadillo-8.400.0.tar.gz
```

Comment:
Hey,
I think there is conflict in this file. Please resolve it. Also, rebase your PR to know more about the conflicts.

Reply:
Sorry @walragatver I should have done this earlier but I did not notice it. I have done with rebasing now thanks for helping.